content/source/docs/enterprise/install/uninstall.html.md

```diff
@@ -20,10 +20,28 @@ After you initiate `uninstall`:
 1. Enter `yes` when asked if you want to continue with the installation.
 
 	```
-	$ sudo ./uninstall.sh
-	This script will completely uninstall Terraform Enterprise and Replicated on this system, as well as remove associated files. This includes the Replicated snapshot directory. Please back up /var/lib/replicated/snapshots if you wish to save them.
+	$ sudo ./uninstall.sh 
+	This script will completely uninstall Terraform Enterprise and Replicated on this system, as well as remove associated files.
 	Do you wish to continue? (y/n)yes
 	Proceeding with uninstall...
+	```
+
+	If there are snapshots present on the system, you will be prompted to move them, proceed with deleting them, or abort.
+	
+	```
+	There appear to be Replicated snapshots stored in /var/lib/replicated/snapshots.
+	1) Move the snapshots to another directory
+	2) Continue uninstall and delete the snapshots
+	3) Cancel the uninstall
+	Select an option: 1
+	Enter the directory to move the snapshots to:/tmp
```

Comment:
```suggestion
	Enter the directory to move the snapshots to: /tmp
```

Reply:
I knew someone was gonna comment on that. I gotta update the script to fix this. (This is all copy-pasta directly from actual script output.